app/components/markdown/markdown.tsx

```diff
@@ -0,0 +1,503 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
```

Comment:
For my own understanding, what happen to this file?

Reply:
Renamed and added the latex inline and code block renderers

Reply:
Oh and converted to functional component from class component